net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/CheckUtil.java

```diff
@@ -0,0 +1,88 @@
+package net.sf.eclipsecs.core.config.meta;
+
+import java.util.Set;
+
+import com.puppycrawl.tools.checkstyle.PackageNamesLoader;
+import com.puppycrawl.tools.checkstyle.PackageObjectFactory;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+import com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck;
+import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
+import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
+
+import net.sf.eclipsecs.core.CheckstylePlugin;
+
+public final class CheckUtil {
+    private CheckUtil() {
+    }
+
+    public static String getAcceptableTokens(String checkName) {
```

Comment:
As you substract required token, please name method `getNodifiableTokens`

Reply:
Done